src/scancode/cli.py

```diff
@@ -223,20 +232,39 @@ def reindex_licenses(ctx, param, value):
 number of files processed. Use --verbose to display file-by-file progress.
 '''
 
+formats = ('json', 'json-pp', 'html', 'html-app', 'spdx-tv', 'spdx-rdf')
+plugin_formats = {}
 
 class ScanCommand(BaseCommand):
     short_usage_help = '''
 Try 'scancode --help' for help on options and arguments.'''
 
+    options = [ op for op in scan_proper_pm.hook.add_cmdline_option() if isinstance(op, click.Option) ]
+
+    def __init__(self, name, context_settings=None, callback=None,
+                 params=None, help=None, epilog=None, short_help=None,
+                 options_metavar='[OPTIONS]', add_help_option=True):
+        BaseCommand.__init__(self, name, context_settings, callback,
+                 params, help, epilog, short_help,
+                 options_metavar, add_help_option)
+        plugins_ret_list = scan_proper_pm.hook.add_format()
```

Comment:
are they returning a list or a tuple of two values?
IMHO either the plugin is well behaved and it returns two values or it is not and you should warp this in a try/catch and bail out loudly with a meaningful error if it does not 
Note that in the case of an output format plugin there can be ONLY one. e.g. we dump only one format out. So I am not sure why you would carry a list there. Also the name would be best as `plugins` rather than `plugins_ret_list`